src/index.html

```diff
@@ -11,7 +11,190 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label class="field">
+          Surname:
+          <input
+          class="data"
+          type="text"
+          name="Surname"
+          required
+          autocomplete="off"
+          >
```

Comment:
Please, fix it for all other inputs
```suggestion
          <input
            class="data"
            type="text"
            name="Surname"
            required
            autocomplete="off"
          >
```